include/curlhandle.h

```diff
@@ -11,8 +11,8 @@ namespace newsboat {
 class CurlHandle {
 private:
 	CURL* h;
-	CurlHandle(const CurlHandle&);
-	CurlHandle& operator=(const CurlHandle&);
+	CurlHandle(const CurlHandle&) = delete;
+	CurlHandle& operator=(const CurlHandle&) = delete;
```

Comment:
While you're at it, it probably makes sense to write explicit move constructor and move assignment operator. I don't think the default move-constructor will zero out the `h`.

Reply:
Looks like I added it a few weeks ago, but didn't push it yet: https://github.com/newsboat/newsboat/pull/1718/commits/254ad9e485773c721e3fe9376402d70fd10d125d